src/psyclone/tests/parse/kernel_test.py

```diff
@@ -664,7 +750,7 @@ def my_init(self, _):
 def test_get_int_array_constructor_err(monkeypatch):
     ''' Check that we raise the appropriate error if we fail to parse the
     array constructor expression. '''
-    from fparser.two import Fortran2003
+
```

Comment:
I would suggest a blank line before the closing quotations instead as in other tests.

Reply:
Done